src/executor/evaluate/mod.rs

```diff
@@ -573,6 +573,22 @@ async fn evaluate_function<'a, T: 'static + Debug>(
             }
         }
         .map(Evaluated::from),
+        Function::Repeat{ expr, num } => {
+            let expr = match eval_to_str(expr).await? {
+                Nullable::Value(v) => v,
+                Nullable::Null => {
+                    return Ok(Evaluated::from(Value::Null));
+                }
+            };
+            let num = match eval_to_integer(num).await? {
+                Nullable::Value(v) => v,
+                Nullable::Null => {
+                    return Ok(Evaluated::from(Value::Null));
+                }
```

Comment:
Let's just have a single more test case.
Precisely, this `Nullable::Null` is not tested by current `repeat` integration test